src/components/load.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import progressbar from '../assets/images/progressbar.png';
+
+const Progress = () => {
+  const randomProgress = (min = 60, max = 85) => (Math.floor(Math.random() * (max - min)) + min);
+  const randomChapter = (min = 1, max = 15) => (Math.floor(Math.random() * (max - min)) + min);
+
+  return (
+    <div className="progress-container">
+      <div className="bar-wrapper">
+        <img src={progressbar} alt="progressbar" />
```

Comment:
- [ ] [ **OPTIONAL** ] You did a great job by implementing the style which match exactly the style provided by the original design from Zeppelin. But, I'd like to suggest you instead of using an image for circular bar progress, you can use the react `circularProgressBar packages` so that the icon could be updated with the pourcentage of your `rendomProgress` function. Have a look on this documentation [react-circular-progressbar](https://www.npmjs.com/package/react-circular-progressbar) to know more about how it works.

Reply:
Hi @RolandM99! 
That's awesome!.. I will check it out for sure :rocket: